server/include/data_store.hpp

```diff
@@ -54,11 +58,16 @@ class DataStore
 
     bool RunQuery(const std::string& sql);
 
-    static int DbCallback(void* not_used, int argc, char** argv, char** azColName); // Not used
+    static int DbCallback(void* not_used, int argc, char** argv, char** azColName);
+
+    bool ReadDistinctMacAddrs(const std::string& device_id, std::vector<std::string>& mac_addrs);
 
     sqlite3*                        database_;
     std::mutex                      database_lock_;
     std::shared_ptr<spdlog::logger> console_;
+
+    // For tests only
+    std::string executing_sql_;
```

Comment:
Can you elaborate more on how this is used and add is as a comment?

Reply:
Done